### DISCUSSION TITLE
Supports for auto-complete and error linting?

### DISCUSSION THREAD

**Discussion by H4ckOm** [cpeditor/cpeditor#728] [Q&A] [answered]
can we use C++ Intelligence on cpeditor just like we have for VS Code?

**Comment by ouuan** [comment, marked as answer]
You can read about [Language Server](https://cpeditor.org/docs/setup/#setup-language-server).

Now we support linting only, we'll add more in the future, including auto-complete.

You can track the progress of auto-complete in #156

**Reply by H4ckOm** [reply]
Thanks for reply, also please tell can I configure own shortcuts like ctrl + c to copy a line and ctrl + x to cut the line?

**Reply by ouuan** [reply]
> Thanks for reply, also please tell can I configure own shortcuts like ctrl + c to copy a line and ctrl + x to cut the line?

You need to open another discussion for a new topic.

**Reply by ouuan** [reply]
BTW, you should mark it as answer: 
![screenshot](https://user-images.githubusercontent.com/30581822/105455219-20470900-5cbe-11eb-8e7d-482545a3fb1e.png)
